apis/config/v1beta1/configuration_types.go

```diff
@@ -349,3 +352,28 @@ type ClusterQueueVisibility struct {
 	// Defaults to 10.
 	MaxCount int32 `json:"maxCount,omitempty"`
 }
+
+// +kubebuilder:validation:Enum=LessThanOrEqualToFinalShare;LessThanInitialShare
```

Comment:
This marker wouldn't work fine since the Config API is not CRD.
So, we need to implement validations here: https://github.com/kubernetes-sigs/kueue/blob/0cdb8fda4bdf07d6a702cf8160acd71e2a07053b/pkg/config/validation.go

Reply:
I was hopeful 😅 
I'll move it.